#cluster with vel

cluster_element[] clst
point3d[] vel
